src/docfx/build/xref/ExternalXrefSpec.cs

```diff
@@ -30,6 +30,9 @@ internal record ExternalXrefSpec : IXrefSpec
 
     FilePath? IXrefSpec.DeclaringFile => null;
 
+    // used for localizable xrefmap
+    public SpecType? SpecType { get; set; }
```

Comment:
Seems `init` only is enough

Reply:
the set statement is used here  https://github.com/freewheel70/docfx/blob/c192e6bf20e31597b1b96850eb3c14ad07bedcb3/src/docfx/build/xref/XrefResolver.cs#L227